### PR TITLE
[DOCS-164] Update Web methodologies page

### PR DESCRIPTION
**Pull request by mjang-cobalt**
- Include more complete info
- Set up progressive disclosure for complete info with expand functionality

@ana-dashuk-cobalt While I'd like your approval, I'm going to wait until we get Marion's review before merging (and proceeding with the other parts of the Jira.

Also, this helps me assess the workload required to convert the 67? Zendesk articles that we want to convert to product docs

### CODE REVIEW

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 1]
```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
...
+information on how you can:
+
+- Remediate each finding
+- Improve your overall security posture
+
+If you remediate findings during the supported pentest period, our pentesters
+can retest your updated components against each discovered vulnerability.
+
+{{% /expand %}}
+
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:217]:
```suggestion
</br>
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 2]
```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
...
+{{% expand "Business and application logic mapping" %}}
+</br>
+
+Pentesters manually examine the target applications to map:
+
+- Business functions
+- Workflows
+- Underlying processes.
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:77]:
```suggestion
- Underlying processes
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 3]
```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
...
+</br>
+
+Our pentesters use both commercial and freeware security tools to assess the targeted
+application. They'll modify these tools as needed, to make sure that scanning can find
+security issues on every segment of your asset, and the application as a whole.
+
+In addition, our pentesters run automated crawls to:
+
+- Identify any pages are available to unauthenticated users
+- Determine the full site tree.
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:103]:
```suggestion
- Determine the full site tree
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 4]
```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
...
+</br>
+
+In this part of the pentest process, our pentesters:
+
+- Use automated tools for web application crawling
+  - Verify the results manually
+- Run manual crawling tests for better coverage
+  - Verify authentication on protected areas of the application
+
+With automated scanning, our pentesters
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:122]:
```suggestion
With automated scanning, our pentesters:
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 5]
```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
...
+- CVE reports or tracked by CVE entries
+
+Our pentesters also consider the workflows and business logic into consideration
+when they identify vulnerabilities in the application.
+
+The assessment includes tests for vulnerabilities such as:
+
+- Injection attacks that probe the robustness of server-validation routines
+- Session management flaws that could allow user impersonation
+- Flaws in access control that expose data or enable users to gain elevated privileges.
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:157]:
```suggestion
- Flaws in access control that expose data or enable users to gain elevated privileges
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 6]
```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
...
+- Access control management
+  - Cross-Origin Resource Sharing (CORS) implementation
+- Vulnerabilities outlined in the OWASP [API Security Project](https://owasp.org/www-project-api-security/)
+
+For each finding, pentesters determine the risk associated with each issue by:
+
+- Demonstrate how the issue could be exploited
+- Evaluate its impact within the context of the business function, data, and
+  users of the asset
+- Set up a Proof-of-Concept exploitation to:
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:176]:
```suggestion
- Demonstrating how the issue could be exploited
- Evaluating its impact within the context of the business function, data, and
  users of the asset
- Setting up a Proof-of-Concept exploitation to:
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 7]
```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
...
+- Burp Suite Pro/Community
+- OWASP ZAP
+- Dirble
+- Nuclei
+{{% /expand %}}
+
+{{% expand "Ongoing assessments" %}}
+</br>
+
+Our pentesters report their findings, in real-time, through the Cobalt platform.
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:191]:
```suggestion
Our pentesters report their findings, in real time, through the Cobalt platform.
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 8]
```diff
@@ -42,16 +42,178 @@ We follow an industry standard methodology primarily based on the OWASP Applicat
...
+
+You're welcome to communicate with our pentesters for each of their findings.
+
+{{% /expand %}}
+
+{{% expand "Report, triage, and retest" %}}
+</br>
+
+Our pentesters report and triage all vulnerabilities during the assessment. You
+can review details of all findings, in real-time, through the Cobalt platform.
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:205]:
```suggestion
can review details of all findings, in real time, through the Cobalt platform.
```

**Review by ana-dashuk-cobalt — APPROVED** [review]
Great article, @mjang-cobalt! Thorough and to the point. I only added typo-level suggestions.

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md (hunk 6 above)
Comment (mjang-cobalt) [reply content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:176]: Yeah, you're right, in context, we should go with the "gerund"

**Comment by mjang-cobalt** [comment]
Note: Approved by CS (Marion Sornette) via internal Slack. Merging.

**Pull request merged**